### DISCUSSION TITLE
i think there is some miss spelling in the changelog

### DISCUSSION THREAD

**Discussion by anispwyn** [ppy/osu#18677] [Q&A] [answered]
not about the game but i literally have no idea where to report this so i will do it here

![IMG_20220613_211131](https://user-images.githubusercontent.com/36318840/173373365-2dd695ad-4932-434e-ac1f-86087aeb33a3.jpg)

and add "sane" frame limiter
i guess the word "sane" is some
or maybe i'm really dumb and dont understand what does "sane" mean

**Comment by bdach** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/20418176/173375011-0e69ed0e-a8df-42ce-a93f-a4469a9afcbc.png)

in this case the usage was intended in the second context

**Reply by anispwyn** [reply]
oh.
urban dictionary arent give me any answer at all
thanks

**Reply by bdach** [reply]
i think you should use better primary sources than the urban dictionary...
